Tests/store_packs.py

```diff
@@ -459,10 +465,59 @@ def upload_integration_images(self, storage_bucket):
             print(f"Uploaded {len(pack_local_images)} images for {self._pack_name} pack.")
         except Exception as e:
             task_status = False
-            print_error(f"Failed to upload {self._pack_name} pack integration images. Additional info\n: {e}")
+            print_error(f"Failed to upload {self._pack_name} pack integration images. Additional info:\n {e}")
         finally:
             return task_status, uploaded_integration_images
 
+    def upload_author_image(self, storage_bucket):
+        """Uploads pack author image to gcs.
+
+        Searches for `Author_image.png` and uploads author image to gcs. In case no such image was found,
+        default Base pack image path is used and it's gcp path is returned.
+
+        Args:
+            storage_bucket (google.cloud.storage.bucket.Bucket): gcs bucket where author image will be uploaded.
+
+        Returns:
+            bool: whether the operation succeeded.
+            str: public gcp path of author image.
+
+        """
+        task_status = True
+        author_image_storage_path = ""
+
+        try:
+            repo_author_image_path = os.path.join(self._pack_repo_path, Pack.AUTHOR_IMAGE_NAME)
+
+            if os.path.exists(repo_author_image_path):
+                image_to_upload_storage_path = os.path.join(STORAGE_BASE_PATH, self._pack_name, Pack.AUTHOR_IMAGE_NAME)
+                pack_author_image_blob = storage_bucket.blob(image_to_upload_storage_path)
+
+                with open(repo_author_image_path, "rb") as author_image_file:
+                    pack_author_image_blob.upload_from_file(author_image_file)
+
+                author_image_storage_path = pack_author_image_blob.name if USE_GCS_RELATIVE_PATH \
+                    else pack_author_image_blob.public_url
+
+                print_color(f"Uploaded successfully {self._pack_name} pack author image", LOG_COLORS.GREEN)
+            else:  # use default Base pack image
+                author_image_storage_path = os.path.join(STORAGE_BASE_PATH, BASE_PACK, Pack.AUTHOR_IMAGE_NAME)
+
+                if not USE_GCS_RELATIVE_PATH:
+                    # disable-secrets-detection-start
+                    author_image_storage_path = os.path.join(GCS_PUBLIC_URL, storage_bucket.name,
+                                                             author_image_storage_path)
+                    # disable-secrets-detection-end
+                print_warning((f"Skipping uploading of {self._pack_name} pack author image "
```

Comment:
I am not sure this should be a warning as it is going to be standard flow.

Reply:
Changed